### PR TITLE
Support AIRR export for sequence and receptor datasets

### PR DESCRIPTION
**Pull request by BlueBasilisk**

### CODE REVIEW

File: source/workflows/instructions/dataset_generation/DatasetGenerationInstruction.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ class DatasetGenerationInstruction(Instruction):
                 - my_generated_dataset
                 - my_dataset_from_adaptive
             export_formats: # list of formats to export the datasets to
-                - AIRR
+                - AIRR # only supported for RepertoireDatasets
```
Comment (pavlovicmilena) [inline source/workflows/instructions/dataset_generation/DatasetGenerationInstruction.py:33]: the comment is not true anymore?

File: source/IO/dataset_export/AIRRExporter.py [hunk 2]
```diff
@@ -26,19 +34,51 @@ class AIRRExporter(DataExporter):
...
 
     @staticmethod
-    def export(dataset: RepertoireDataset, path, region_type=RegionType.IMGT_CDR3):
-        if not isinstance(dataset, RepertoireDataset):
-            logging.warning(f"AIRRExporter: dataset {dataset.name} is a {type(dataset).__name__}, but only repertoire dataset export is currently "
-                            f"supported for AIRR format.")
-        else:
-            PathBuilder.build(path)
+    def export(dataset: Dataset, path, region_type=RegionType.IMGT_CDR3):
+        # if not isinstance(dataset, RepertoireDataset):
```
Comment (pavlovicmilena) [inline source/IO/dataset_export/AIRRExporter.py:38]: we don't need this anymore?

File: source/IO/dataset_export/AIRRExporter.py [hunk 3]
```diff
@@ -16,6 +23,7 @@
 class AIRRExporter(DataExporter):
     """
     Exports a RepertoireDataset of Repertoires in AIRR format.
+    This exporter does not support Sequence- or ReceptorDatasets
```
Comment (pavlovicmilena) [inline source/IO/dataset_export/AIRRExporter.py:26]: It supports?

**Comment by BlueBasilisk** [comment]
Thanks! I removed the unnecessary comments now :)

**Review by pavlovicmilena — APPROVED** [review]

**Pull request merged**